src/spicy/spicyz/glue-compiler.h

```diff
@@ -162,6 +163,18 @@ class GlueCompiler {
     /** Destructor. */
     virtual ~GlueCompiler();
 
+    /**
+     * Returns the AST context in use. Only available once the driver has
+     * initialized the glue compiler.
+     */
+    auto context() { return _driver->context()->astContext().get(); }
+
+    /**
+     * Returns the AST builder in use. Only available once the driver has
+     * initialized the glue compiler.
+     */
+    auto builder() { return static_cast<::spicy::Builder*>(_driver->builder()); }
```

Comment:
Nit: This made me briefly wonder whether `context` or `builder` returned values instead of references. Specifying `auto*` (or slightly weird here: `auto&`) instead of `auto` would have made that clearer for me 🤷